go.mod

```diff
@@ -137,4 +137,4 @@ require (
 	sigs.k8s.io/json v0.0.0-20211020170558-c049b76a60c6 // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.1 // indirect
 	sigs.k8s.io/yaml v1.2.0 // indirect
-)
+)
```

Comment:
i saw this change on multiple files, is it wanted?

Reply:
No, they are not, let me revert this change, sorry